src/Tribe/Featured_Events/Permalinks_Helper.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @internal
+ */
+class Tribe__Events__Featured_Events__Permalinks_Helper {
+	public function hook_up() {
```

Comment:
I've seen:

* `hooks()`
* `add_hooks()`
* `hook()`

This PR uses `hook_up()`. It'd be sweet to have a standard name for this sort of thing. I'm partial to `hooks()` because it is short, but it is a plural noun, which is weird. Maybe `hook_up()` is better.

Thoughts, @brian, @bordoni, @lucatume, and @barryhughes?